backend/src/routes/api/images/imageUtils.ts

```diff
@@ -240,9 +240,12 @@ const mapImageStreamToBYONImage = (is: ImageStream): BYONImage => ({
   imported_time: is.metadata.creationTimestamp,
   url: is.metadata.annotations['opendatahub.io/notebook-image-url'],
   provider: is.metadata.annotations['opendatahub.io/notebook-image-creator'],
-  recommendedAcceleratorIdentifiers: jsonParseRecommendedAcceleratorIdentifiers(
+  recommendedAcceleratorIdentifiers: jsonParseRecommendedIdentifiers(
     is.metadata.annotations['opendatahub.io/recommended-accelerators'],
   ),
+  recommendedHardwareProfileIdentifiers: jsonParseRecommendedIdentifiers(
+    is.metadata.annotations['opendatahub.io/recommended-hardware-profile-identifiers'],
```

Comment:
i think we should reuse `opendatahub.io/recommended-accelerators` just becuase it is already being used in the imagestreams that are ootb. might be easier it keep it for now. in the future we should change it to something more generic like recommended resource identifiers